plugin-base/build.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
 //    pluginCompile "com.github.martinkyov.Xchange:xchange-coinbase:136289732f459f489caa0cd4860f6dcd3b76712f"
 //    pluginCompile "com.github.martinkyov.Xchange:xchange-coinbase:e55e846c55711644eec463b40d2a484e2092fbec"
 //    pluginCompile "com.github.martinkyov.Xchange:xchange-coinbase:874ff9e8e4f86964368d5bff8969b35f9e2dc557"
-    pluginCompile "com.github.martinkyov.Xchange:xchange-coinbase:87ec0906a19e21bca9770dd2d787b451df76dba7"
+    pluginCompile "com.github.Slithercze.Xchange:xchange-coinbase:1169721f1018552adc4dc8f1b5fc8be6ceca9eef"
```

Comment:
Finally we use  the object "CoinbaseShowTransactionV2" and you dont need the new version.